backend/bdd/step_definitions/test_dataroom_collaboration.py

```
@@ -0,0 +1,341 @@
+
+
+@then(parsers.parse('both the owner and "{email}" should see "{link_name}" in the dataroom share links'))
+def verify_share_links_visible(user_context, email, link_name):
+    room = user_context['dataroom']
+    owner_client = user_context['api_client']
+    collab_client = APIClient()
+    collab_client.force_authenticate(user=user_context['collab_user'])
+
+    r1 = owner_client.get(f'/api/v1/share-links/?dataroom_id={room.id}')
+    assert r1.status_code == status.HTTP_200_OK
+    names1 = [l['name'] for l in r1.data]
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)